### ISSUE TITLE
Create a demo on Hugging Face Hub

### ISSUE THREAD

**Issue by adeenayakup** [deepseek-ai/DreamCraft3D#3]
Hi!

Very cool work! It would be nice to create a demo on the [Hugging Face Hub](https://huggingface.co/spaces)!

This is a [step-by-step guide](https://huggingface.co/docs/hub/spaces-sdks-gradio) explaining the process in case you're interested.

Please let us know if you would be interested and if you have any questions. 😊

**Comment by MrTornado24** [comment]
Hello, thank you for your interest! We are currently working on enhancing training speed and aim to develop a demo as you suggested in the near future!

**Comment by yosun** [comment]
or replicate.com

**Comment by MavenDE** [comment]
Would this code work on 16GB VRAM 4060 ti ? There is 20GB VRAM written on the main page and that excludes anything beside 4090.

**Comment by york-yu-ctw** [comment]
it will never happens, since it cost 20GB of VRAM and even the fast process consume 40 mins to generated first mesh

**Comment by Treata11** [comment]
> it will never happens, since it cost 20GB of VRAM and even the fast process consumes 40 mins to generate first mesh

@york-yu-ctw 
40 mins? Have you personally tried it?

**Comment by Treata11** [comment]
> Hello, thank you for your interest! We are currently working on enhancing training speed and aim to develop a demo as you suggested in the near future!

Well @MrTornado24, **nine** months have passed! Any good news coming soon?

**Comment by yosun** [comment]
yes esp since 5s AI3D has been SOTA for the last 6 months - surely the amazing team behind deepseek can do 500ms AI3D?
